README.md

```diff
@@ -73,6 +73,8 @@ While you can just contribute translations through this repository, an easier wa
 
 If you're interested in chatting about our translation efforts, join our [i18n room on Matrix](https://matrix.to/#/#dade-i18n:matrix.altpeter.me).
 
+Notice to translators: When adding a new language, the `slug` of the `generator.md` page must not be translated, as our redirection code depends on it.
```

Comment:
I think https://github.com/datenanfragen/website/blob/master/src/i18n/README.md would be a better place for this.

The root README only has information on how to contribute translations, but no specific guidelines.